GOVERNANCE.adoc

```diff
@@ -0,0 +1,71 @@
+
+=== Become a Maintainer
+Making contributions does not require becoming a maintainer, or obtaining commit access to the ServiceTalk
+project/organization. However if you are interested in taking a more formal role with decision making power you should
+consider becoming a Maintainer. Some characteristics of a Maintainer are as follows:
+
+* Be involved in contributing code, pull request review, triage of issues and addressing user questions in one or more
+forums such as Github, Stackoverflow, etc.
+* Maintain sustained contribution to the ServiceTalk project and spend a reasonable amount of time on it.
+* Show deep understanding of the areas contributed to, and good consideration of various reliability, usability,
+backward compatibility, and performance requirements.
+
+If you believe you satisfy these above criteria please open a
+link:https://github.com/apple/servicetalk/compare[github pull request] modifying the
+xref:MAINTAINERS.adoc[Maintainers doc] describing at least 5 non-trivial pull requests that have
+been accepted without major modifications. Please also make your case as to why you feel you will need Maintainer status
+moving forward and intention of continued involvement. Existing Maintainers must vote on the PR and a majority vote is
+required to merge and add the new Maintainer (see <<Decision Making and Voting>>).
+
+=== Expectations of a Maintainer
+After becoming a Maintainer you have the following expectations:
+
+* You are granted commit-after-approval to all parts of ServiceTalk.
+* You may commit an obvious change without first getting approval. The community expects you to use good judgment.
+Examples are reverting obviously broken patches, correcting code comments, and other minor changes.
+* You are allowed to commit changes without approval to the portions of ServiceTalk to which you have contributed or
+for which you have been assigned responsibility. Such commits must not break the build and follow backward compatibility
+requirements. This is a “trust but verify” policy, and commits of this nature are reviewed after being committed.
+
+Even with commit access, your changes are still subject to code review. Of course, you are also encouraged to review
+other peoples’ changes.
+
+=== Losing Maintainer Status
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to
+be moved to emeritus status. If possible, try to complete your work or help find someone to pick up your work before
+stepping down. If a maintainer has stopped contributing for a reasonable amount of time, other maintainers may propose
+to move such maintainers to emeritus list without prior notice. The PR for a such as change would serve as the notice.
+Such a PR should have @mention of the maintainer in question and should remain open for at least a period of two weeks.
+Any disagreements will be resolved by a vote of the maintainers per the voting process above.
```

Comment:
I wonder if we should also add something for how someone who is `emeritus` can become an active maintainer again... Often foundations etc have special rules for something like this which are more lightweight then the initial process of becoming a maintainer.

Reply:
lets start with the existing process and amend if this becomes more common in the future.